scripts/less2sass.js

```diff
@@ -91,9 +92,35 @@ Less2Sass.prototype.convertInterpolatedVariables = function() {
 
 Less2Sass.prototype.convertVariables = function() {
     // Matches any @ that doesn't have 'media ' or 'import ' after it.
-    var atRegex = /@(?!(media|import|mixin|font-face|keyframes)(\s|\())/g;
-
-    this.file = this.file.replace(atRegex, '$');
+    //
+    // SIMPLE LINE ICONS DEVIATION:
+    // Additionally, we capture the variable name and full variable declaration
+    // for "simple-line-icons"-specific logic for creating default variables
+    // (because LESS doesn't have the concept of default variable declarations).
+    //
+    // For example, if we have a LESS variable that looked like:
+    //
+    //      @simple-line-font-path: "../fonts/";
+    //
+    // The "varDeclaration" group would be 'simple-line-font-path: "../fonts/"' and the varName
+    // group would be "simple-line-font-path".
+    var atRegex = /@(?!(media|import|mixin|font-face|keyframes)(\s|\())(?<varDeclaration>(?<varName>([^:]*))[^;]*)/g;
```

Comment:
I'm no RegEx expert, but I used regex101.com to validate that I was selecting what I desired:

<img src="https://user-images.githubusercontent.com/971647/89320714-73150180-d647-11ea-99d4-bf31fd906ed8.png" width="500px" />

As mentioned in the comment, the `varDeclaration` and `varName` capturing groups are the additions made to run the below logic of defaulting only the variables that we want to be overridable.

Reply:
What if we add `// default` comment and match default variables by this comment? https://regex101.com/r/zsu4Ba/1
![Screenshot_20200805_101524](https://user-images.githubusercontent.com/2629882/89388699-abc6e080-d704-11ea-9d86-6fb6a082e7bf.png)

Reply:
That sounds like a good idea to me! I'll update this weekend.